common/logisticspipes/gui/GuiGuideBook.java

```diff
@@ -0,0 +1,171 @@
+package logisticspipes.gui;
+
+import java.io.IOException;
+
+import net.minecraft.client.gui.GuiButton;
+import net.minecraft.client.gui.GuiScreen;
+import net.minecraft.client.gui.GuiSlider;
+import net.minecraft.client.gui.inventory.GuiContainer;
+import net.minecraft.client.renderer.GlStateManager;
+import net.minecraft.util.ResourceLocation;
+
+import org.lwjgl.Sys;
+
+import logisticspipes.LPConstants;
+import logisticspipes.utils.string.StringUtils;
+
+public class GuiGuideBook extends GuiScreen {
+
+	private static final ResourceLocation GUI_BOOK_TEXTURE = new ResourceLocation(LPConstants.LP_MOD_ID, "textures/gui/guide_book.png");
+
+	private GuiButton nextPageBtn, prevPageBtn, slider;
+	private boolean dragging = false;
+	private int mouseX, mouseY;
+
+	//Book vars
+	private int PAGE = 1;
+	private int PAGE_MAX = 10;
+	private int MAX_LENGTH = 50;
+
+	public GuiGuideBook() {
+		super();
+	}
+
+	/** Draw a background container sized based on x, and y size. This is not 1:1 scale.*/
+	protected void drawGuiBackgroundLayer(int x, int y) {
+		GlStateManager.color(1.0F, 1.0F, 1.0F, 1.0F);
+		mc.renderEngine.bindTexture(GUI_BOOK_TEXTURE);
+		int xRem = x;
+		int yRem = y;
+		int eight = 8;
+		GlStateManager.pushMatrix();
+		GlStateManager.translate(0.0F, 0.0F, -2.0F);
+		this.drawDefaultBackground();
+		GlStateManager.popMatrix();
+		GlStateManager.pushMatrix();
+		GlStateManager.translate(-x*4, -y*4, -1.0F);
+		/**This is the darker background like the inventory has. Check whether this is wanted or not.*/
+		GlStateManager.translate(this.width / 2, this.height / 2, 0.0F);
+		/**TopLeft*/
+		drawTexturedModalRect(0, 0, 0, 0, eight * 4, eight * 4);
+		xRem -= 4;
```

Comment:
Instead of doing this here, just set `xRem = x - 8` and `yRem = y - 8` initially?

Reply:
Yes it is the same and I just did it this way as a reminder as to why I have to do it at all